munimap/static/js/digitize/digitize-popup-controller.js

```diff
@@ -8,42 +8,45 @@ angular.module('munimapDigitize')
             $scope.featureFilter = featureFilter;
             $scope.needsRefresh = false;
             $scope.openDigitizePopupFor = undefined;
-            $scope.lastOpenedPopup = undefined;
+            $scope.featureEditorProps = {};
```

Comment:
couldn't the default value be `undefined`? Then `Object.keys($scope.featureEditorProps).length === 0` would not be necessary as well.

Reply:
Yes, but then we would have to check for undefined in `digitize-popup.html`. Unfortunately, we cannot use the optional chaining operator there, so we would have to write something like

```html
<div
  anol-feature-properties-editor="featureEditorProps? featureEditorProps.feature : undefined"
  layer="featureEditorProps? featureEditorProps.layer : undefined"
  template-url="munimap/feature-properties-editor.html"
></div>
```

which I wanted to avoid.